bin/deploy

```diff
@@ -36,5 +36,14 @@ aws s3 sync \
 
 echo "Deployed styleguide to http://rcl.policygenius.com/$VERSION/index.html"
 
+echo '+++ Deploying styleguide latest to S3'
+aws s3 sync \
+  ./styleguide s3://rcl-pg/latest/ \
```

Comment:
@j-clark My only concern is that I might have to force the overwriting to this directory. I'm not sure ~